wiki/Beatmap/id.md

```diff
@@ -1,12 +1,118 @@
 ---
-outdated: true
-outdated_since: f4d914b0b826e19e50caeea90f58557065e9e0b7
+tags:
+  - mapset
+  - beatmapset
 ---
 
 # Beatmap
 
-*Artikel ini tidak mencakup seputar [Tingkat Kesulitan Beatmap](/wiki/Beatmap/Difficulty) secara mendalam.*
+Sebuah **beatmap** (terkadang disebut sebagai *beatmapset*) adalah sebuat set dari level gim ([tingkat kesulitan](#difficulty)) yang terdiri dari berbagai [hit object](/wiki/Hit_object) dan hampir selalu mewakili satu lagu. Beatmap juga mencakup komponen lainnya, semua dikemas dalam bentuk arsip dengan [ekstensi .osz](/wiki/osu!_File_Formats/Osz_(file_format)):
 
-**Beatmap** merupakan istilah yang merujuk pada level-level permainan yang tersedia untuk dimainkan. Tiap-tiap berkas beatmap mengandung berbagai informasi yang terkait dengan aspek-aspek yang hadir di dalam permainan seperti [*hit object*](/wiki/Hit_Objects), [*timing*](/wiki/Beatmap_Editor/Timing), metadata, pengaturan tingkat kesulitan *(difficulty setting)*, efek-efek visual, dan lain sebagainya.
+- Lagu itu sendiri, disimpan dalam format MP3 atau Ogg.
```

Comment:
in the English article this list is being written in all lowercase so consider changing `Lagu itu sendiri` to i.e. `berkas lagu itu sendiri, yang tersimpan dalam format MP3 atau Ogg` :

![image](https://user-images.githubusercontent.com/36564236/113480554-ff214680-94be-11eb-815a-42c5cafebe8f.png)